### PR TITLE
feat(DEE-1103): added dark theme to feedback web component

### PR DESCRIPTION
**Pull request by yashoswalyo**
# Closes/Fixes/Resolves

### 

# Explain the feature/fix

Introduced a dark theme mode for feedback component.

[Screencast from 2024-06-10 18-13-16.webm](https://github.com/1-Platform/op-components/assets/42431299/6b3e143e-48bf-4723-8cd9-6c1c059eff4b)

[Screencast from 2024-06-10 18-14-13.webm](https://github.com/1-Platform/op-components/assets/42431299/088f3ae6-6972-4f31-92ce-a04f978bb64e)

## Does this PR introduce a breaking change

No

## Screenshots

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] My changes generate no new warnings
- [x] I have performed a self-review of my own code
- [ ] I have documented my code, particularly in areas like todo, complex logic, quick fix, temporary patch, etc.
#### If it is a new component
- [ ] Does your component follow One Platform style guidelines?
- [ ] Does your component web accessibility standards? [Helper Doc](https://www.w3.org/TR/wai-aria-1.1/)
- [ ] Does your component support desktop screen sizes (350px, 720px, 1150px ,1920px)
#### Browsers you have tested in
- [ ] Latest two versions of Mozilla Firefox and Google Chrome supported by Red Hat Enterprise Linux distribution
- [x] Google Chrome [Latest 2 versions]
- [x] Mozilla Firefox [Latest 2 versions]
- [ ] Microsoft Edge [Latest 2 versions]
- [ ] Apple Safari [Latest 2 versions]
- [ ] Microsoft Internet Explorer 11

### CODE REVIEW

**Comment by yashoswalyo** [comment]
@hybridx @deshmukhmayur @riginoommen needs review

**Comment by yashoswalyo** [comment]
Tests and Build is passing

image

File: packages/opc-feedback/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@one-platform/opc-feedback",
-  "version": "0.1.1-alpha",
+  "version": "0.2.0",
```
Comment (hybridx) [inline packages/opc-feedback/package.json:3]: I suppose it is still in alpha state

**Review by hybridx — APPROVED** [review]

**Comment by hybridx** [comment]
LGTM approved 🎉

File: packages/opc-feedback/package.json (hunk 1 above)
Comment (yashoswalyo) [reply packages/opc-feedback/package.json]: Yes added alpha tag

**Review by hybridx — APPROVED** [review]

**Comment by hybridx** [comment]
@yashoswalyo please resolve the conflicts

**Pull request merged**